### DISCUSSION TITLE
Reminder: Hack submissions are due tonight!

### DISCUSSION THREAD

**Discussion by pamelafox** [microsoft/AI-Chat-App-Hack#112] [Announcements]
Submit your project anytime before February 12th 23:59 PM PST:

You may submit your project here when it's ready: 🚀 [Project Submission](https://github.com/microsoft/AI-Chat-App-Hack/issues/new?assignees=&labels=&projects=&template=project.yml&title=Project%3A+%3Cshort+description%3E)

Check out this video for step by step project submission guidance: [Project Submission Video](https://github.com/microsoft/hack-together-teams/assets/3199282/572ea387-61ec-4b77-9885-23b5b2bd39bd)

If your project isn't ready for submission quite yet but you're interested in having it showcased this afternoon, please drop a link/video/screenshot below. And then submit it tonight!

**Comment by crissins** [comment]
is there an ETA to announce the winners?

**Reply by pamelafox** [reply]
We hope to have judging complete in the next 2 weeks. It takes a fair bit of time for the judges to review each submission (it took me about a day) so we need that time for the judging. Thanks for your patience!
